### ISSUE TITLE
Cannot run on Ubuntu 16.04

### ISSUE THREAD

**Issue by eleidan** [silentsokolov/tocenter#1]
```
$ ./tocenter 
./tocenter: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by ./tocenter)

$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
```

**Comment by silentsokolov** [comment]
Try it. I built it with musl. [tocenter-0.1.1-linux-musl.tar.gz](https://github.com/silentsokolov/tocenter/files/3865256/tocenter-0.1.1-linux-musl.tar.gz)

**Comment by eleidan** [comment]
Here what I get:
```
$ ./tocenter
bash: ./tocenter: No such file or directory
```
BTW, should my system meet some dependencies?
I think Musl build is good solution, generally.

**Comment by silentsokolov** [comment]
Yes, static linking is failing. This is probably caused by some specific crate that links to a C library. I'll fix it (maybe ...).

But, I compiled a version for the ubuntu 16.04 
[tocenter-0.1.1-ubuntu-16.04.tar.gz](https://github.com/silentsokolov/tocenter/files/3865734/tocenter-0.1.1-ubuntu-16.04.tar.gz)

I know that it is not a solution to the problem, but at least something
